app/assets/stylesheets/application.css.scss

```diff
@@ -42,6 +42,10 @@ body {
   font-size: 1.5em;
 }
 
+#chart > svg {
+  padding: 30px 0px 30px 0px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `30px 0px` instead of `30px 0px 30px 0px`<br>`0px` should be written without units as `0`